src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,21 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+    LocalDate byWhen;
+    public Deadline(String description, LocalDate byWhen, boolean done) {
```

Comment:
Might want to consider naming the boolean "done" something that sounds like a boolean according to the Java coding standard provided. For example, we can name the boolean variable "isDone" instead of "done". :)

Reply:
Yep, will add that in. Thanks for catching this :)